.pre-commit-config.yaml

```diff
@@ -15,36 +15,46 @@
 default_language_version:
   python: python3.8
 
+ci:
+  autofix_prs: true
+  autoupdate_commit_msg: '[pre-commit.ci] pre-commit suggestions'
+  # submodules: true
+
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v2.3.0
+    rev: v3.4.0
     hooks:
-      - id: trailing-whitespace
       - id: end-of-file-fixer
+      - id: trailing-whitespace
+      #- id: check-yaml
+      - id: pretty-format-json
 
-  - repo: local
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.8.0
     hooks:
       - id: isort
         name: imports
         entry: isort
-        args: [--settings-path, ./pyproject.toml]
-        language: system
+        language: python
         types: [python]
```

Comment:
I don't think we need to annotate:

```
        language: python
        types: [python]
```

But not 100% sure.

Same for the other hooks

Reply:
some required it, probably not all...